Github-Org-Management-Policy.md

```diff
@@ -0,0 +1,50 @@
+# Node.js GitHub Organization Management Policy
+
+The Node.js Foundation Github Organization (https://github.com/nodejs) is
+provided as a development resource by the Node.js Foundation under the
+operational direction of the Node.js Technical Steering Committee (TSC).
+
+## Organization Roles
+
+Individual users within a GitHub organization may be granted one of three
+distinct roles: Member, Owner, or Billing manager. The permissions granted
+to each role are documented in the GitHub documentated [Permission levels for
+an organization](https://help.github.com/articles/permission-levels-for-an-organization/).
+
+### Billing manager
+
+All Billing managers for the Node.js Foundation GitHub Organization shall
+be determined by the Node.js Foundation Board of Directors with notification
+given to the TSC.
+
+### Owners
+
+All voting members of the Node.js TSC shall have Owner permissions within the
+Node.js Foundation GitHub Organization. Additional users may be granted Owner
+permissions at the discretion of the TSC through normal TSC motion and simple
+majority vote.
+
+### Members
+
+GitHub users may be granted Member status in the Node.js Foundation GitHub
+Organization by any Owner.
+
+## Repositories
+
+Any organization member may create new repositories within the Node.js
+Foundation GitHub Organization. Any such repository is considered to be a
+project under the ownership of the Node.js Foundation, and thereby subject
+to the Intellectual Property and Governance policies of the Foundation under
+the operational direction of the TSC.
```

Comment:
I read this as all repos are automatically under the TSC, however in practice we have some repos that are not under the TSC (community, board, etc). How does this play in to the scope of the TSC? What if one of these new repos doesn't fit into that scope?

I wonder if it would make more sense to have them default to falling under the board, and then officially brought under the TSC via some sort of explicit process (vote in a meeting?) so that we can vet them for scope.

@williamkapke I'm especially curious to hear your thoughts on this.